docs-src/docs/dataset-exploration/uploading-predictions.md

```diff
@@ -56,6 +58,23 @@ except requests.exceptions.RequestException as e:
     print(f"Request error: {e}. {msg}")
 ```
 
+Predictions groups connected to segmentation datasets require one extra parameter called `classMapping`. This is used to map the class names 
+in the predictions to the class names in the annotations. The `classMapping` parameter is a list of dictionaries, where each dictionary
+contains the keys `annotated` and `predicted`. The `annotated` key is the class name in the annotations, and the `predicted` key is the 
+class name in the predictions. All class names in the predictions and the annotations must be present in the class mappings, even if they 
+don't need to be mapped.
+
+```python
+example_body = {
+    "name": "My predictions group",
+    "description": "A description of my new predictions group",
+    "classMapping": [
+        {"annotated": "oak", "predicted": "tree"},
+        {"annotated": "only_in_annotations"}
```

Comment:
Also add something like
`{"annotated": "_background", "predicted": "not_tree"}`

Reply:
Why should there be an underscore before background?